examples/wallet/app/utils/mnemonic.js

```diff
@@ -0,0 +1,33 @@
+import { validateMnemonic, mnemonicToSeedSync } from 'bip39';
+
+import wordlist from './wordlist.en';
+
+export const isValidMnemonic = (mnemonicPhrase: string, numberOfWords = 12) => {
```

Comment:
why is it necessary to receive the number of words as a parameter?

Reply:
Deleted, it was not necessary